src/server/string_family.cc

```diff
@@ -191,13 +193,26 @@ OpResult<bool> ExtendOrSkip(const OpArgs& op_args, string_view key, string_view
   return ExtendExisting(op_args, *it_res, key, val, prepend);
 }
 
-OpResult<string> OpGet(const OpArgs& op_args, string_view key) {
+OpResult<string> OpGet(const OpArgs& op_args, string_view key, bool del_hit = false) {
   OpResult<PrimeIterator> it_res = op_args.shard->db_slice().Find(op_args.db_cntx, key, OBJ_STRING);
   if (!it_res.ok())
     return it_res.status();
 
   const PrimeValue& pv = it_res.value()->second;
 
+  if (del_hit) {
+    string key_bearer = GetString(op_args.shard, pv);
+
+    DVLOG(1) << "Del: " << key;
+    auto& db_slice = op_args.shard->db_slice();
+
+    auto res = db_slice.Del(op_args.db_cntx.db_index, it_res.value());
```

Comment:
I guess it must find it here, so maybe use CHECK macro instead of checking the result.

Reply:
Fixed all 👍